src/screens/SearchScreen/SearchScreen.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import SearchScreenContainer from './components/SearchScreenContainer';
+
+const SearchScreen = ({navigation}) => {
+  return <SearchScreenContainer />;
```

Comment:
We talked about this today and I think perhaps one small change you could make is to just rename `SearchScreenContainer` => `SearchContainer` (giving `SearchScreen` the responsibility to handle _screen_ related concerns, and leaving the `SearchContainer` to think about data/state related to the search).